src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -85,6 +92,24 @@ const SubsidiesSummary = ({
   const hasAvailableSubsidyOrRequests = (
     hasActiveLicenseOrLicenseRequest || hasAssignedCodesOrCodeRequests || learnerCreditSummaryCardData
   );
+  const [assignmentOnlyLearner, setAssignmentOnlyLearner] = useState(false);
+  useEffect(() => {
+    const assignmentsData = redeemableLearnerCreditPolicies?.flatMap(item => item?.learnerContentAssignments || []);
+    const { hasActiveAssignments } = getActiveAssignments(assignmentsData);
+    if (
+      !hasActiveLicenseOrLicenseRequest
+      && !hasAssignedCodesOrCodeRequests
+      && !hasCurrentEnterpriseOffers
+      && hasActiveAssignments
+    ) {
+      setAssignmentOnlyLearner(true);
+    }
+  }, [
```

Comment:
[question] @irfanuddinahmad Actually, one more clarifying question here... if a learner has an auto-applied (non-assignable) budget available, will this logic take that into account? For example, this conditional does not take into account any non-assignable `redeemableLearnerCreditPolicies` the learner may have access to.

Reply:
@adamstankiewicz you are correct. assignments are a subset within hasAvailableLearnerCreditPolicies ... any more pointers on how to check for this? :)

Reply:
@irfanuddinahmad I believe checking whether there are any non-assignable `redeemableLearnerCreditPolicies` should be sufficient?:

```jsx
const hasAutoAppliedLearnerCreditPolicies = redeemableLearnerCreditPolicies?.filter(policy => policy.policyType !== 'AssignedLearnerCreditAccessPolicy').length > 0;
```

And then include the check against `!hasAutoAppliedLearnerCreditPolicies` in your conditional above on line 99 🤔

Reply:
@irfanuddinahmad [inform] Another [PR](https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/896) recently merged with a similar fix in the `isDisableCourseSearch` function to take into account auto-applied (non-assignable) budgets similar to this suggestion.

Note that it introduces a `POLICY_TYPES` constant.